gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/PubSubRpc.java

```diff
@@ -69,4 +70,6 @@
   Future<PullResponse> pull(PullRequest request);
 
   Future<Empty> modify(ModifyPushConfigRequest request);
+
+  ScheduledExecutorService executor();
```

Comment:
Probably fine for now, but we should think how we can avoid thread-pools proliferation when using multiple gcloud services.